src/module.ts

```diff
@@ -0,0 +1,41 @@
+import { defineNuxtModule } from '@nuxt/kit'
+import { join } from 'pathe'
+import { createFeedTemplate, resolveSources, writeFeedFile } from './feed'
+import type { FeedSource, FeedSourcesFactory } from './feed'
+
+export interface ModuleOptions {
+  sources: FeedSource[] | FeedSourcesFactory
+}
+
+export default defineNuxtModule<ModuleOptions>({
+  meta: {
+    name: '@nuxtjs/feed',
+    configKey: 'feed',
+    compatibility: {
+      nuxt: '^3.0.0',
+      bridge: false
+    }
```

Comment:
For fast upgrade, let me drop Bridge support at least for now.

Reply:
I think dropping bridge support here makes sense. If there is demand, I'd go for a later adoption.